sources/pmwebapi_test.go

```diff
@@ -55,49 +55,28 @@ func TestUnmarshal(t *testing.T) {
 	}
 }
 
-func TestLabelTypes(t *testing.T) {
-	units := ""
-	pcpType := ""
-	name := "kal"
-	response := typeLabel(units, pcpType, name)
-	expected := "kal"
-	if response != expected {
-		t.Fatalf("Failed when units and type strings were empty. Expected: %s, Got: %s", expected, response)
-	}
-
-	units = ""
-	pcpType = ""
-	name = ""
-	response = typeLabel(units, pcpType, name)
-	expected = ""
-	if response != expected {
-		t.Fatalf("Failed when all strings were empty. Expected: %s, Got: %s", expected, response)
-	}
+func TestWeirdMetrics(t *testing.T) {
 
-	units = "kal"
-	pcpType = "CoUnTeR"
-	name = "kal"
-	response = typeLabel(units, pcpType, name)
-	expected = "kal_kal_total"
-	if response != expected {
-		t.Fatalf("Failed when type was case sensitive. Expected: %s, Got: %s", expected, response)
+	name := []string{"count / sec", "Kbyte", "nanosec", "millisec", "failcnt", "byte", "megabytess", "Mbyte", "sec", "bytes_byte", "kilobytess"}
+	expected := []string{"count_per_second", "kilobytes", "nanoseconds", "milliseconds", "failcount", "bytes", "megabytes", "megabytes", "seconds", "bytes", "kilobytes"}
+	for n, _ := range name {
+		if fixNaming(name[n]) != expected[n] {
+			t.Fatalf("Failed when name was %s. Expected: %s, Got: %s", name[n], expected[n], fixNaming(name[n]))
+		}
+		n++
```

Comment:
Is this necessary? If you tried to access `name[n]`, or `expected[n]` after this line you will get a panic.